examples/interfaces/interfaces.go

```diff
@@ -1,19 +1,19 @@
-// _Interfaces_ are named collections of method
-// signatures.
+// Le _Interfacce_ in Go sono collezioni di 
+// signature di metodi.
```

Comment:
Ho trovato su Wikipedia che <a href="https://it.wikipedia.org/wiki/Firma_(programmazione)">Firma</a> è la traduzione di signature anche in questo caso. Lo teniamo con signature o lo italianizziamo?